docs/getting-started/tutorial-load-dataset.asciidoc

```diff
@@ -151,13 +151,13 @@ The accounts data set doesn't require any mappings, so at this point we're ready
 {ref}/docs-bulk.html[`bulk`] API to load the data sets with the following commands:
 
 [source,shell]
-curl -H 'Content-Type: application/x-ndjson' -XPOST 'localhost:9200/bank/account/_bulk?pretty' --data-binary @accounts.json
-curl -H 'Content-Type: application/x-ndjson' -XPOST 'localhost:9200/shakespeare/doc/_bulk?pretty' --data-binary @shakespeare_6.0.json
-curl -H 'Content-Type: application/x-ndjson' -XPOST 'localhost:9200/_bulk?pretty' --data-binary @logs.jsonl
+curl -u elastic -H 'Content-Type: application/x-ndjson' -XPOST ‘<elasticsearch-URL or localhost>:<port>/<index-name>/<type>_bulk?pretty’ --data-binary @<filename>
```

Comment:
- The quotes here are the stylized single quotes, which can break things. `‘’` versus `''`
- It is missing a `/` after `<type>` and before `_bulk`.
- Can we just change the host to match the other examples?
    - `<host>`

```
# <host> Elasticsearch listens at localhost by default
# <port> Elasticsearch HTTP Port defaults to 9200
curl -u elastic -H 'Content-Type: application/x-ndjson' -XPOST '<host>:<port>/<index-name>/<type>/_bulk?pretty' --data-binary @<filename>
```

Reply:
I agree, I like `<host>` over `<elasticsearch-URL or localhost>`

Reply:
Do you think we should leave the port as 9200?

Reply:
I think so. Honestly would be okay leaving the host as `localhost` too.